server/test/unit/server/managers/repo/test_distributor.py

```diff
@@ -3,13 +3,12 @@
 from .... import base
```

Comment:
a lot removed here by mocking the creation of a repo. also, testing when a repo does not exist is no longer necessary because that is covered by the custom repo query set tests.